wgpu-hal/src/gles/adapter.rs

```diff
@@ -291,6 +291,10 @@ impl super::Adapter {
             wgt::Features::ADDRESS_MODE_CLAMP_TO_BORDER,
             extensions.contains("GL_EXT_texture_border_clamp"),
         );
+        features.set(
+            wgt::Features::ADDRESS_MODE_CLAMP_TO_ZERO,
```

Comment:
let's combine it with the case above? Just put the `wgt::Features::ADDRESS_MODE_CLAMP_TO_BORDER | wgt::Features::ADDRESS_MODE_CLAMP_TO_ZERO`